data/ui/AboutDialog.ui.in

```diff
@@ -19,6 +19,7 @@ Contributors:
 	Dan Callaghan &lt;djc@djc.id.au&gt;
     </property>
     <property name="artists">Peter Hutterer &lt;peter.hutterer@who-t.net&gt;</property>
```

Comment:
completely unrelated to this commit but calling me an artist after looking at the quality of the mousetrap is pushing the term a bit :)

Reply:
Just accept the compliment ;)